src/main/java/com/epam/ta/reportportal/dao/ProjectRepositoryCustomImpl.java

```diff
@@ -132,4 +136,29 @@ public ProjectInfo findProjectInfoFromDate(Long projectId, LocalDateTime fromDat
 	public List<String> findAllProjectNames() {
 		return dsl.select(PROJECT.NAME).from(PROJECT).fetchInto(String.class);
 	}
+
+	@Override
+	public Page<Project> findAllIdsAndProjectAttributes(ProjectAttributeEnum projectAttribute, Pageable pageable) {
```

Comment:
could be done using filters after the new query builder

Reply:
done